src/components/AllBooks/allBooks.js

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+import { Container } from "./allBooks.styles";
+import { Card } from "./allBooks.styles";
+import { Genre } from "./allBooks.styles";
+
+export const AllBooks = (props) => {
+  const mappedBooks = props.data.map((book) => {
+    return (
+      <Card>
+        <Link to={`/book/${book.volumeInfo.industryIdentifiers[0].identifier}`}>
```

Comment:
```suggestion
        <Link to={`/book/${book.volumeInfo.industryIdentifiers[0].identifier}`}>
```
maybe  store this in a const